cif_core.dic

```diff
@@ -25109,21 +25109,142 @@ save_atom_type_scat.versus_stol_list
          '_atom_type_scat_versus_stol_list'
          '_atom_type.scat_versus_stol_list'
 
-    _definition.update            2013-04-17
+    _definition.update            2023-04-10
     _description.text
 ;
-    List of scattering factors as a function of sin theta on lambda.
-    List has the form [[<stol value 1> <scatfac 1>] [<stol value 2> <scatfac
-    2>] ....] in increments of 0.01, increasing from 0.0.
+    A table of scattering factors as a function of sin theta over lambda. This
+    table should be well-commented to indicate the items present. Regularly
+    formatted lists are strongly recommended.
 ;
     _name.category_id             atom_type_scat
     _name.object_id               versus_stol_list
+    _type.purpose                 Describe
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_ATOM_TYPE_SCAT_TABLE
+
+    _definition.id                ATOM_TYPE_SCAT_TABLE
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-04-10
+    _description.text
+;
+    The CATEGORY of data items used to list atomic scattering factor values for
+    use in crystallographic structure studies.
+;
+    _name.category_id             ATOM_TYPE
+    _name.object_id               ATOM_TYPE_SCAT_TABLE
+
```

Comment:
@jamesrhester, can we have `ATOM_TYPE` as the parent category here? Since both the parent (`ATOM_TYPE`) and the child (`ATOM_TYPE_SCAT_TABLE`) categories are looped, wouldn't it also mean that they can be represented in a single loop? Clearly, in this case it is not possible since a single atom type can be associated with multiple entries in the `ATOM_TYPE_SCAT_TABLE`.

Or is the single-loop representation automatically rejected in this case based on the fact that `ATOM_TYPE_SCAT_TABLE` has a composite key only one part of which is linked to `ATOM_TYPE`  category key?

Reply:
The parent-child relationship for looped categories means that they could be joined on their key data names, which must all be common. Therefore you are right, these categories can't have a formal parent-child relationship.

Reply:
Ok, I changed the parent category to ATOM.